Backend/Remora.Discord.API.Abstractions/API/Rest/IDiscordRestApplicationAPI.cs

```diff
@@ -195,10 +201,11 @@ Task<Result<IApplicationCommand>> CreateGuildApplicationCommandAsync
         string name,
         string description,
         Optional<IReadOnlyList<IApplicationCommandOption>> options = default,
-        Optional<bool> defaultPermission = default,
         Optional<ApplicationCommandType> type = default,
         Optional<IReadOnlyDictionary<string, string>?> nameLocalizations = default,
         Optional<IReadOnlyDictionary<string, string>?> descriptionLocalizations = default,
+        Optional<IDiscordPermissionSet> defaultMemberPermissions = default,
+        Optional<bool?> dmPermission = default,
```

Comment:
dm_permission is not available for guild commands (see discord/discord-api-docs#4861). Remove this parameter.